eng/scripts/Logging-Functions.ps1

```diff
@@ -20,6 +27,9 @@ function LogErrorForFile($file, $errorString)
   if (Test-SupportsDevOpsLogging) {
     Write-Host ("##vso[task.logissue type=error;sourcepath=$file;linenumber=1;columnnumber=1;]$errorString" -replace "`n", "%0D%0A")
   }
+  elseif (Test-SupportsGitHubLogging) {
+    Write-Host ("::error file=$file,line=1,col=1::$errorString" -replace "`n", "%0D%0A")
```

Comment:
It is cool that GH actions support these logging commands as well. They look to support a few more we might want to take advantage of at some point as well like the summary https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-adding-a-job-summary.

Reply:
I agree the ability to generate a custom job summary by writing markdown to a file on disk (via an env var) is very powerful.